master/getting-started/rkt/tutorials/basic.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Calico networking with rkt
+---
+
+
+This tutorial describes how to set up a Calico cluster in a pure rkt environment
+using CNI with Calico specific network and IPAM drivers.
+
+Using the Calico CNI plugin, the required network setup and configuration
+for networking containers using Calico is handled automatically as part of the
+standard network and container lifecycle.  Provided the network is created
+using the Calico driver, creating a container using that network will
+automatically add the container to the Calico network, creating all necessary
+Calico configuration and setting up the interface and routes in the container
+accordingly.
+
+The Calico IPAM driver must be used in addition to the the Calico network
+driver.  This provides IP address management using the configured Calico IP
+Pools as address pools for the container, preferentially selecting sub-blocks
+of IPs for a particular host.
+
+## 1. Environment setup
+
+To run through the worked example in this tutorial you will need to set up two hosts
+with a number of installation dependencies.
```

Comment:
_that meet the following requirements_

Then we should probably lay out the requirements that it must meet in bulleted form :)

Then we can have a "to quickly set up an env that meets these reqs, try the vagrant tutorial".

WDYT?